package.json

```diff
@@ -50,13 +49,13 @@
     "karma-mocha-reporter": "^2.2.5",
     "karma-sourcemap-loader": "^0.3.8",
     "karma-webpack": "^5.0.0",
-    "mock-socket": "^9.0.3",
+    "mock-socket": "^9.1.0",
```

Comment:
This is necessary in order to import Client from mock-socket https://github.com/onsip/SIP.js/pull/972/files#diff-c8df3bcb89d2bae5907ec3711283b7c2aa02cf080f6c4d610fa61bc8e2075b0bR3